source/language/gates.rst

```diff
@@ -163,6 +163,8 @@ and applies the controlled gate
   0 & 0 & e^{-i\tau/2} & 0 \\
   0 & 0 & 0 & e^{i\tau/2} \end{array}\right).
 
+``gphase(γ)`` may optionally take qubit arguments to apply the phase directly to the qubit. This is
+typically used in conjunction with a gate modifier such as control.
```

Comment:
I see that you need that in order to parse the `p` gate in `stdgates.inc`. But, I wonder if that wasn't actually a typo, as I am not sure what it means to apply a global phase to a qubit.

Reply:
Addressed via controlled global phase gate language. See https://github.com/Qiskit/openqasm/pull/203/commits/4471686f9e8f72a249b267a99622e1a86150777b.